src/telemetry/metrics/ws/mod.rs

```diff
@@ -111,8 +142,18 @@ pub fn record_rpc(cx: &TelemetryContext, res_size: usize, is_error: bool) {
 		]);
 		duration = cx.start.elapsed().as_millis() as u64;
 		req_size = cx.size as u64;
+	} else if let Some(cx) = cx.get::<NotificationContext>() {
+		attrs.extend_from_slice(&[
+			KeyValue::new("rpc.method", "notification"),
+			KeyValue::new("rpc.error", is_error),
+			KeyValue::new("rpc.live_id", cx.live_id.clone()),
+			KeyValue::new("rpc.notification_id", cx.notification_id.clone()),
+		]);
 	} else {
 		// If a bug causes the RequestContent to be empty, we still want to record the metrics to avoid a silent failure.
+		if true {
+			panic!("lasagna!");
```

Comment:
```suggestion
			panic!("Burger! 🍔");
```

Reply:
Rejected. Bring it up with the Chief Greek Officer @dimitrianoudi

Reply:
I needed to remove that line, but lets agree on using Moussaka in the future